src/app/(proper_react)/(onboarding)/user/welcome/GetStarted.tsx

```diff
@@ -59,9 +60,9 @@ export const GetStarted = (props: Props) => {
               }
               illustration={<Image src={howItWorksHero} alt="" />}
             >
-              <div className={styles.dialogContents}>
-                <ol className={styles.howItWorksList}>
-                  <li className={styles.howItWorksEntry}>
+              <div className={`${viewStyles.dialogContents}`}>
```

Comment:
nit: not sure if this _needs_ the template string or not vs just inlining it like L64-65.
```suggestion
              <div className={viewStyles.dialogContents}>
```